src/node/RosTCP.js

```diff
@@ -21,6 +24,9 @@ function RosTCP(options) {
       port: options.port
     });
   }
+  if(options.http || options.socketio){
+    this.io = new SocketIO(options, this);
```

Comment:
Going to run into the same issues as #126 and https://github.com/RobotWebTools/rosbridge_suite/issues/140